### ISSUE TITLE
Are you still maintaining this?

### ISSUE THREAD

**Issue by svenstaro** [not-fl3/glium_text_rusttype#13]

**Comment by not-fl3** [comment]
I am back here :) 

Not going to actively work on this in the near future, but will be there for PRs and issues. 
I see your PR, its great, if it will help you - I could add you to crates.io owners etc, just let me know :)

**Comment by svenstaro** [comment]
I don't really need to be a maintainer if you are still ok maintaining this. I would just appreciate it if you made tags for releases and in fact made a release for my recent changes.

**Comment by svenstaro** [comment]
Also you should throw this up on travis and activate weekly autobuilds just to check whether things keep working.

**Comment by not-fl3** [comment]
Published your changes to crates.io. 
Will keep this issue opened until I manage to set up travis.

**Comment by svenstaro** [comment]
Nice stuff, appreciate it.

**Comment by Boscop** [comment]
@not-fl3 Could you please update deps, e.g. glium to 0.28? :)

Btw, the table is broken in the docs: https://docs.rs/glium_text_rusttype/0.3.3/glium_text_rusttype/

And Rust supports inclusive ranges now:
> As of now, Rust doesn't provide a way to create inclusive ranges yet so thou shall add +1 to the outer bound.

**Comment by not-fl3** [comment]
Not really, I moved from glium to miniquad https://github.com/not-fl3/glium_text_rusttype/pull/17#issuecomment-622983702 and not using this anymore. 

I can merge pull requests etc, but if someone wants to properly maintain - would be happy to give write access/transfer crate rights or whatever is needed :)

**Comment by swiftcoder** [comment]
Sent PR #18 to bump the glium version, as it's broken past glium 0.26 at the moment. Happy to help out herding PRs/releases through if you like, since I'm making use of this crate.
